cassis/cas.py

```diff
@@ -171,6 +177,40 @@ def __init__(self, typesystem: TypeSystem):
 class Cas:
     """A CAS object is a container for text (sofa) and annotations"""
 
+    NAME_SPACE_UIMA_CAS = "uima" + TypeSystem.NAMESPACE_SEPARATOR + "cas"
```

Comment:
Are these not already in the typesystem.py ?

Reply:
Not as constants, just as lists. Also, in the Apache UIMA Java SDK, these constants are in the Cas, so that's where I look for them ;)

Reply:
It would then be better to put them in the typesystem and let the lists use them there. For me they are more in the typesystem and not in the cas, as the former is where cassis uses them the most

Reply:
Can the type system not import them from the CAS? I am be bit biased because I am used to the constants being in the CAS.

Reply:
The CAS in cassis is mostly not using it, only the type system and serializer/deserializer do I think to I find it strange to put it in the cas.py where it is mostly not used and let others import it.